card/lib/cardio.rb

```diff
@@ -23,10 +22,11 @@ def cache
     end
     
     def set_config config
-      @@config = config
-      @@root = @@config.root
-        
-      config.autoload_paths += Dir["#{Cardio.gem_root}/mod/*/lib/**/"]
+      @@config, @@root = config, config.root
+
+      config.autoload_paths += Dir["#{gem_root}/mod/*/lib/**/"]
+      config.autoload_paths += Dir["#{gem_root}/lib/**/"]
```

Comment:
cool. I'm tempted to limit this to lib/* and lib/card/**/ so that we don't look through generators for autoload stuff.  But cool that this solves issues.

Does this make it so that we don't really need the require_dependency in engine.rb?